src/GitExtensionsVSIX/GitExtensionsVSIX.csproj

```diff
@@ -1,7 +1,6 @@
 ﻿<Project>
```

Comment:
I thought about it, but decided not to do it, because the older extension is usable by VS 2015, VS 2017 and VS 2019. hence appending version number is not good for existing project